switchtec.c

```diff
@@ -1295,6 +1363,10 @@ static int switchtec_pci_probe(struct pci_dev *pdev,
 		  &stdev->mmio_part_cfg->mrpc_comp_hdr);
 	enable_link_state_events(stdev);
 
+	if (stdev->dma_mrpc){
```

Comment:
Style nit: Single line if statements should not have parenthesis.

Reply:
fixed, thx.